NEWS.md

```diff
@@ -83,6 +83,8 @@ Library improvements
 
   * The `chop` and `chomp` functions now return a `SubString` ([#18339]).
 
+  * Numbered stackframes printed in stacktraces can be opened in an editor by entering the corresponding number in the REPL and pressing `^Q`.
```

Comment:
don't have to run news-update right away as long as the reference is in the right format for it